Tests/iaas/entropy/entropy-check.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python3
+"""Entropy checker
+
+Check given cloud for conformance with SCS standard regarding
+entropy, to be found under /Standards/scs-0101-v1-entropy.md
+
+Return code is 0 precisely when it could be verified that the standard is satisfied.
+Otherwise the return code is the number of errors that occurred (up to 127 due to OS
+restrictions); for further information, see the log messages on various channels:
+    CRITICAL  for problems preventing the test to complete,
+    ERROR     for violations of requirements,
+    INFO      for violations of recommendations,
+    DEBUG     for background information and problems that don't hinder the test.
+"""
+from collections import Counter
+import getopt
+import logging
+import os
+import re
+import sys
+import tempfile
+import time
+import warnings
+
+import fabric
+import invoke
+import openstack
+import openstack.cloud
+
+
+logger = logging.getLogger(__name__)
+
+# use telling names here, maybe even reuse the security group in case of a leak?
```

Comment:
It should no longer be a problem, but it could happen that the security group created by the script was not properly removed, so it got leaked. And I thought if we found an existing security group from a previous run of the script we could just reuse that one instead of creating a new one. But as I said, it's no longer an issue, so I will just remove the comment.

Reply:
Resolved with 9249d4f777adaf82e1a56023a3f21197f65cfd18.